ZadanieDomowe7/Loops.cs

```diff
@@ -27,15 +28,27 @@ static public int Power(int num1, int num2)
             return result;
         }
 
-        static public List<int> RemainderOfDivisionOnNumber(int num1)
+        public static int[] CountRemainderOfDivisionOnNumber(int num)
         {
-            List<int> numbers = new List<int>();
+            int[] numbers;
+            int count = 0;
 
-            for (int i = num1; i < 10; i++)
+            for (int i = num; i < 10; i++)
             {
-                if (i % num1 == 0)
+                if (i % num == 0)
                 {
-                    numbers.Add(i);
+                    ++count;
+                }
+            }
+
+            numbers = new int[count];
+            count = 0;
```

Comment:
Re-usage is a good practice but not here I think. Please create `j` for numbers indexer